src/main/resources/resource/WebGui/app/service/views/LocalSpeechGui.html

```diff
@@ -2,15 +2,19 @@
     <div class="col-md-4">
         <br/>
         <br/>
-        type <br/>
+        <b>type</b> <br/>
         {{service.config.speechType}}
+        
         <select ng-model="service.config.speechType" ng-change="setType(service.config.speechType)" class="form-control" title="select a type of local voice control">
             <option ng-repeat="type in service.types" ng-value="{{type}}">{{type}}</option>
         </select>
-
-        
+        <div ng-show="service.config.speechType === 'Pico2Wav'">
+            <b>Install pico2wave</b><br/>
+            https://doc.ubuntu-fr.org/svoxpico<br/>
+            <pre>sudo apt-get install libttspico-utils</pre>
```

Comment:
apt-get is considered legacy, most systems recommend plain `apt`

Reply:
i'm dating myself .. ok will change